documentation/join-devnet-gui.md

```diff
@@ -1,26 +1,25 @@
-# Setup Lotus Devnet using a GUI
+# Join Lotus Devnet
```

Comment:
You can't join the devnet using the POND UI. It set's up a separate local network, and for the most part is just a developer tool for debugging networks. We had it in the readme because it's a shows nicely what really happens under the hood

Reply:
Okay thank you for this clarification, sorry about the misunderstanding.